control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1632,6 +1632,13 @@ func (r *HostedControlPlaneReconciler) reconcileKubeAPIServer(ctx context.Contex
 	}); err != nil {
 		return fmt.Errorf("failed to reconcile api server deployment: %w", err)
 	}
+
+	// Ensure all replicas are updated and ready
+	// https://kubernetes.io/releases/version-skew-policy/#supported-component-upgrade-order
```

Comment:
can you clarify in the comment which part of the skew policy exactly is this enforcing?